.github/workflows/snyk.yml

```diff
@@ -0,0 +1,18 @@
+name: Code Scanning with Snyk
+on: push
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/golang@master
+        continue-on-error: true # To make sure that SARIF upload gets called
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
does this `secret` is org level or we have to set repo level?

Reply:
I set one at the repo level in github.com/cfergeau/admin-helper, hopefully it can be done at the org level in crc-org

Reply:
I dropped the snyk commit as this was too problematic.